web/source/keymanweb.js

```diff
@@ -4289,7 +4322,7 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
       if(document.selection  &&  !keymanweb.legacy)
         util.attachDOMEvent(document, 'selectionchange', keymanweb._SelectionChange);
     
-      // Restore and reload the currently selected keyboard 
+      // Restore and reload the currently selected keyboard, selecting a default keyboard if necessary.
       keymanweb.restoreCurrentKeyboard(); 
```

Comment:
This line triggers a load of the previous keyboard state from the KMW cookie within 'native' mode KMW.